doc/UserGuide.md

```diff
@@ -193,6 +193,15 @@ see the Mserialize documentation on [Adapting custom types][mserialize-act] and
 [mserialize-act]: Mserialize.html#adapting-custom-types
 [mserialize-rec]: Mserialize.html#adapting-user-defined-recursive-types-for-visitation
 
+## Logging Standard Types
+
+`std::chrono::system_clock::time_point` objects can be logged after adoption by including a header file:
```

Comment:
Do we want to add a caveat that this might break on some weird systems pre C++20 (`std::chrono::system_clock` epoch wasn't standardised on the unix epoch till then)? Does this caveat apply to all binlog line timestamps also?

Reply:
Yes, it applies to both kind of timestamps, and the assumption is tested in TestTime.cpp

Reply:
Added a paragraph to limitations to make this clear.